modules/integration/src/test/scala/scala/cli/integration/BspTestDefinitions.scala

```diff
@@ -303,25 +303,24 @@ abstract class BspTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  def runScalaCli(args: String*) = os.proc(TestUtil.cli, args)
+
   def testScalaTermination(
     currentBloopVersion: String,
     expectedBloopVersionAfterScalaCliRun: String
-  ): Unit =
-    TestUtil.retry() {
```

Comment:
Let's keep the `retry`? (And add it to the new tests too, just in case?)

Reply:
sorry, lost it during rebase